tests/baselines/reference/binaryIntegerLiteralError.errors.txt

```diff
@@ -16,13 +13,7 @@ tests/cases/conformance/es6/binaryAndOctalIntegerLiteral/binaryIntegerLiteralErr
     
     var obj1 = {
         0b11010: "hi",
-        ~~~~~~~
-!!! error TS2300: Duplicate identifier '0b11010'.
         26: "Hello",
-        ~~
-!!! error TS2300: Duplicate identifier '26'.
         "26": "world",

```

Comment:
this we should keep. it is an error under strict mode to have duplicate properties in an object literal.